src/configs/localip.rs

```diff
@@ -0,0 +1,23 @@
+use crate::config::ModuleConfig;
+
+use serde::Serialize;
+use starship_module_config_derive::ModuleConfig;
+
+#[derive(Clone, ModuleConfig, Serialize)]
+pub struct LocalipConfig<'a> {
+    pub ssh_only: bool,
+    pub format: &'a str,
+    pub style: &'a str,
+    pub disabled: bool,
+}
+
+impl<'a> Default for LocalipConfig<'a> {
+    fn default() -> Self {
+        LocalipConfig {
+            ssh_only: true,
+            format: "@[$localip]($style)",
```

Comment:
Is there a reason to have the `@` in here, or did that get copied from the example configuration?

Reply:
Also, please insert a space at the end of the format string--otherwise subsequent modules get screwed up

```suggestion
            format: "[$localip]($style) ",
```